### DISCUSSION TITLE
通知一下各位, 这个版本iOS授权码出现了问题。 我们提出了两个解决方案。

### DISCUSSION THREAD

**Discussion by MaxMKMax** [getlantern/lantern-forum-cn#255] [Announcements]
现在我们有2个解决方案。 

1: 您可以尝试重复请求授权并检查所有垃圾邮件/垃圾邮件以查看是否有授权代码。

2: 是我们正在准备新版本，该版本将修复 IOS 上的授权码问题。请耐心等待我们下一个版本的发布。它很快就会问世。我们现在准备着与大家分享所有新版本。感谢您的等待。

对于IOS和Mac用户，我们将为大家提供因问题造成的补偿。我们将延长您的专业版时间，并为该计划安排更长的持续时间。给您带来的不便，敬请谅解。

**Comment by 3173757283** [comment]
我的邮箱：1204703263@qq.com
付款凭据见下图
![pay](https://github.com/getlantern/lantern-forum-cn/assets/47740191/5e2b1510-2064-40e9-ad99-984a27b0f7fb)

**Reply by 3173757283** [reply]
请尽快给我发验证码，我这边急用 感谢

**Reply by MaxMKMax** [reply]
@3173757283 您好。好的，我马上帮您转达给我们的负责人。但很抱歉，团队处于离线状态。他们将于今晚按照美国时区上线。我现在就通知他们，并帮助您今晚再次和他们跟进。会尽快帮您解决。抱歉让您久等了。谢谢!

**Comment by sabrinareng** [comment]
账号：ssr789@163.com
付款凭证：
![Uploading IMG_7391.PNG…]()



请尽快发送验证码给我，感谢！

**Reply by sabrinareng** [reply]
![IMG_7391](https://github.com/getlantern/lantern-forum-cn/assets/163874792/cfd951da-a235-4202-ab0f-b748c008f904)

**Reply by MaxMKMax** [reply]
@sabrinareng 您好, 请您稍等! 我们在研究是否能手动发出授权码. 或者通过别的方法. 不好意思. 请等等.

**Reply by sabrinareng** [reply]
请尽快解决！！！！都一年了哥～

**Comment by AvatarVirgo** [comment]
账号：yuanjie.lin@msn.com，IOS无法登陆专业版无法授权，请尽快解决，谢谢！

**Reply by MaxMKMax** [reply]
@AvatarVirgo 您好, 如果您是受到授权码问题影响的IOS用户，我们将通过重新延长您的专业版使用时间来补偿您。
现在iOS系统暂时无法登陆. 我们正在弄新版本了, 很快可以更新了. 请您耐心等待. 非常抱歉影响了您的体验感!

**Comment by AvatarVirgo** [comment]
@Mike777Khoo
